pkg/env/certificateRefresher_test.go

```diff
@@ -265,8 +280,7 @@ func TestRefreshingCertificate(t *testing.T) {
 				return
 			}
 
-			time.Sleep(test.sleep)
-
+			start <- struct{}{}
 			testkey, testCerts := refreshing.GetCertificates()
```

Comment:
@petrkotas Oh! I think I know why tests flake sometime (https://github.com/Azure/ARO-RP/pull/1691#pullrequestreview-735136798)! We wait before we start creating fake ticks (`start <- struct{}{})`), but we do not wait for `refreshingCertificate` to do the job before we call `GetCertificates`. I think this is the reason for flakes in this PR.